programs/ssl/ssl_client2.c

```diff
@@ -198,7 +199,8 @@ int main(void)
 
 #if defined(MBEDTLS_SSL_SESSION_TICKETS)
 #define USAGE_TICKETS                                       \
-    "    tickets=%%d          default: 1 (enabled)\n"
+    "    tickets=%%d              default: 1 (enabled)\n"    \
+    "    new_session_tickets=%%d  default: 1 (enabled)\n"
```

Comment:
Minor: as a matter of principle, options in `ssl_client2` should default to "library default", that is, not calling the corresponding `conf` function at all. That's the only way to test the the library defaults are what we think they are.

(Not a blocker for the release, as we've agreed on manual testing for now anyway, but wanted to point it out for future reference.)

Reply:
Otherwise I would have to enable the new session tickets in all ssl-opt.sh resumption tests. Not changed yet but I can if you think it is worth it.

Reply:
Not now, let's leave that to a follow-up.